enterprise/models.py

```diff
@@ -118,6 +118,13 @@ class Meta:
         )
     )
 
+    contact_email = models.EmailField(
```

Comment:
@mattdrayer Are you anticipating we'll need to store more customer contact information ?  If not, then I think a separate model is overkill.

But if so, then maybe this should be `learner_contact_email` instead, to differentiate it from our internal contact email addresses, which we might use for support or otherwise.